site-src/implementations.md

```diff
@@ -0,0 +1,95 @@
+# Implementations
+
+This document tracks downstream implementations of Gateway API and provides status and resource references for them.
+
+Implementors of Gateway API are encouraged to update this document with status information about their implementations, the versions they cover, and documentation to help users get started.
+
+## Implementation Status
+
+|                | [Contour][1]       | [GCP][2]           | [Istio][3]         | [Kong][4]         | [Knative][5]       | [Traefik][6]       |
+| :------------- | :----------------: | :----------------: | :----------------: | :---------------: | :----------------: | :----------------: |
+| [v0.1.x][v0.1] | :heavy_minus_sign: | :heavy_minus_sign: | :heavy_check_mark: |:heavy_minus_sign: | :heavy_minus_sign: | :heavy_check_mark: |
+| [v0.2.x][v0.2] | :heavy_check_mark: | :heavy_plus_sign:  | :heavy_plus_sign:  |:heavy_plus_sign:  | :heavy_minus_sign: | :heavy_plus_sign:  |
+
+- :heavy_check_mark: available
+- :heavy_plus_sign: in progress
+- :heavy_minus_sign: not planned
+
+[1]:#contour
+[2]:#google-cloud-platform
+[3]:#istio
+[4]:#kong
+[5]:#knative
+[6]:#traefik
+
+[v0.1]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0
+[v0.2]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.2.0
+
+## Project References
+
+In this section you will find specific links to blog posts, documentation and other Gateway API references for specific implementations.
+
+### Contour
+
+[Contour][contour] is an open source ingress controller for Kubernetes.
+
+Contour currently implements the latest `v0.2.x` Gateway API Specification.
+
+See the [Contour Gateway API Guide][contour-1] for information on how to deploy and use Contours's Gateway implementation.
+
+[contour]:https://projectcontour.io
+[latest]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+[contour-1]:https://projectcontour.io/guides/gateway-api/
+
+### Google Cloud Platform
+
+The [Google Cloud Platform (GCP)][gcp] is a cloud computing platform and infrastructure provider.
```

Comment:
@robscott is that how you and your team want to be represented? Please do feel free to make direct edits to get it the way you want it.

Reply:
I think "Google Cloud" is the preferred term now. I'll let @mark-church weigh in on what we'd like to include here. I think we'll likely also want to add some GKE specific reference, but not quite sure how to integrate that.

Reply:
```suggestion
The [Google Cloud][gcp] is a cloud computing platform and infrastructure provider.
```

Reply:
@shaneutt thanks for contributing this!

Here's a short statement we can use:

```
The [Google Kubernetes Engine (GKE)][gke] is a managed Kubernetes platform offered by Google Cloud. 

GKE is actively working towards an implementation of the Gateway API for orchestration of [Google Cloud Load Balancing][gclb]. The GKE Gateway Controller will become a managed component of GKE to automate and ease load balancing for containers.

[gke]:https://cloud.google.com/kubernetes-engine
[gclb]:https://cloud.google.com/load-balancingThe [Google Kubernetes Engine (GKE)][gke] is a managed Kubernetes platform offered by Google Cloud. 
```

Reply:
Sounds good, thank you! I've updated to match your contents: 267f6fbfea56821cbaeb1241c1842333abfb5ac9